### ISSUE TITLE
Add Aliases to Help Open Documentation

### ISSUE THREAD

**Issue by DarioS** [Bioconductor/BiocIO#4]
Could we have some aliases such as `\alias{BiocFile}` added to Rd files for all the S4 classes to avoid

```r
> ?BiocFile
No documentation for ‘BiocFile’ in specified packages and libraries
```

**Comment by LiNk-NY** [comment]
Yes, thanks for reporting.

**Comment by LiNk-NY** [comment]
Closed in 89b741e89a83751c4321830cf9f1b3ccea23d47b
